loops-and-arrays.js

```diff
@@ -12,8 +12,9 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i] + 'je');
```

Comment:
ik zou de names [i] binnen de for-loop  alleen aanpassen en pas na de loop  loggen .
dus: names[i] = names [i] + 'je";
 en na de }
console.log(names);